CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ The Product Changelog at **[piwik.org/changelog](http://piwik.org/changelog)** l
 ### Breaking Changes
 * New config setting `enable_plugin_upload` lets you enable uploading and installing a Piwik plugin ZIP file by a Super User. This used to be enabled by default, but it is now disabled by default now for security reasons.
 * New Report class property `Report::$supportsFlatten` lets you define if a report supports flattening (defaults to `true`). If set to `false` it will also set `ViewDataTable\Config::$show_flatten_table` to `false`
+* The event `Live.getAllVisitorDetails` has been deprecated and will be removed in Piwik 4. Use a `VisitorDetails` class instead (see Live plugin). 
```

Comment:
* Let's add a test for this deprecation in tests/PHPUnit/Unit/DeprecatedMethodsTest.php